illegal_whitespace/illegal_whitespace.sh

```diff
@@ -27,7 +27,8 @@ countfile=$WORKSPACE/illegal_whitespace_counters_$gitbranch.csv
 mincountfile=$WORKSPACE/illegal_whitespace_mincounter_$gitbranch.csv
 
 # Co to proper gitdir and gitpath
-cd $gitdir && git reset --hard $gitbranch
+cd $gitdir
```

Comment:
Doesn't this need quoting? Like the original patch was doing with all the rest of (worspace?) occurrences? In fact the test is failing because of the missing quotes, isn't it?